apps/src/code-studio/teacher.js

```diff
@@ -37,4 +43,70 @@ export function onReady() {
       location.reload();
     });
   });
+
+  setStageLockedText();
+}
+
+/**
+ * Query the server for lock status of this teacher's students
+ * @returns {Promise} when finished
+ */
+function queryLockStatus(store, scriptId) {
+  return new Promise((resolve, reject) => {
+    $.ajax(
```

Comment:
When are we going to stop using jQuery.ajax and start using a [fetch polyfill](https://github.com/github/fetch)?